### PR TITLE
Rephrase and tweak `ls -l` discussion

### LINKED ISSUE #24: Consider renaming "columns" in "Scripts, variables, and loops" episode
In "Scripts, variables, and loops" episode, consider using the term "section" instead of "column" for the text underneath "That’s a huge amount of output. Let’s see if we can understand what it is, working left to right."

Because in the existing text, a "column" is made up of several columns of text.

e.g. a time last modified such as `Jan 16 19:17` is referred to as column 5, whereas I would argue this is 3 columns.

I don't want to just do a PR on this because I think it needs to be discussed first.

### PR DESCRIPTION
**Pull request by tkphd**
- Substitute "field" for "column"
- Insert description of References field
- Remove stray '.' characters
- Replace itemized list with enumeration, remove hand-coded field numbering

Closes #24.

### CODE REVIEW

File: _episodes/05-scripts.md [hunk 1]
```diff
@@ -83,41 +83,44 @@ $ ls -l
...
-  indicated in the first permissions column.
-+ **3rd column - Group:** This is the user group of the user who owns the file. Members of this user
-  group have permissions indicated in the second permissions column.
-+ **4th column - Size of file:** This is the size of a file in bytes, or the number of
-  files/subdirectories if we are looking at a directory. (We can use the `-h` option here to get a
-  human-readable file size in megabytes, gigabytes, etc.)
-+ **5th column - Time last modified:** This is the last time the file was modified.
-+ **6th column - Filename:** This is the filename.
+That's a huge amount of output: a full listing of everything in the directory. Let's see if we can
+understand what each field represents, working left to right.
```
Comment (psteinb) [inline _episodes/05-scripts.md]:
```suggestion
understand **what each field of a given row represents**, working left to right.
```

File: _episodes/05-scripts.md [hunk 2]
```diff
@@ -356,7 +359,7 @@ word_counts.txt
 ```
 {: .output}
 
-There's actually even a shortcut to run on all files of a particular type, say all .gz files:
+There's actually even a shortcut to run on all files of a particular type, say all `.gz` files:
```
Comment (psteinb) [inline _episodes/05-scripts.md]: Too many fill words for my taste ;-)
```suggestion
There's a shortcut to run on all files of a particular type, say all `.gz` files:
```

File: _episodes/05-scripts.md [hunk 3]
```diff
@@ -83,41 +83,44 @@ $ ls -l
...
-  files/subdirectories if we are looking at a directory. (We can use the `-h` option here to get a
-  human-readable file size in megabytes, gigabytes, etc.)
-+ **5th column - Time last modified:** This is the last time the file was modified.
-+ **6th column - Filename:** This is the filename.
+That's a huge amount of output: a full listing of everything in the directory. Let's see if we can
+understand what each field represents, working left to right.
+
+1. **Permissions:** On the very left side, there is a string of the characters `d`, `r`, `w`, `x`,
+   and `-`. The `d` indicates if something is a directory (there is a `-` in that spot if it is not
+   a directory). The other `r`, `w`, `x` bits indicates permission to **R**ead, **W**rite, and
```
Comment (psteinb) [inline _episodes/05-scripts.md]:
```suggestion
   a directory). The other `r`, `w`, `x` bits indicate permission to **R**ead, **W**rite, and
```
Not sure about this one as I am a non-native speaker. But a plural plus a 3rd person in singular sounds wrong, doesn't it? Happy to be lectured on this one.

File: _episodes/05-scripts.md [hunk 4]
```diff
@@ -83,41 +83,44 @@ $ ls -l
...
+   user is missing a permission to do something, it's indicated by a `-`.
+     + The first set of `rwx` are the permissions that the owner has (in this case the owner is
+       `yourUsername`).
+     + The second set of `rwx`s are permissions that other members of the owner's group share (in
+       this case, the group is named `tc001`).
+     + The third set of `rwx`s are permissions that anyone else with access to this computer can do
+       with a file. Though files are typically created with read permissions for everyone, typically
+       the permissions on your home directory prevent others from being able to access the file in
+       the first place.
+2. **References:** This counts the number of references (hard links) to the item (file, folder,
```
Comment (psteinb) [inline _episodes/05-scripts.md]: This is a tricky line as it introduces a lot of technical terms. 
```suggestion
2. **References:** This counts the number of references ([hard links](https://en.wikipedia.org/wiki/Hard_link)) to the item (file, folder,
```

File: _episodes/05-scripts.md [hunk 5]
```diff
@@ -83,41 +83,44 @@ $ ls -l
...
+   symbolic link or "shortcut").
```
Comment (psteinb) [inline _episodes/05-scripts.md:114]: What does "shortcut" refer to here?

File: _episodes/05-scripts.md [hunk 6]
```diff
@@ -83,41 +83,44 @@ $ ls -l
...
+3. **Owner:** This is the username of the user who owns the file. Their permissions are indicated
+   in the first permissions field.
+4. **Group:** This is the user group of the user who owns the file. Members of this user group have
+   permissions indicated in the second permissions field.
+5. **Size of file:** This is the size of a file in bytes, or the number of files/subdirectories if
```
Comment (psteinb) [inline _episodes/05-scripts.md]: The claim with reference to folders irritates me. Here is an example of a synthetic folder:
```
$ tree .       
.
├── bar
├── baz.log
└── foo
    ├── baz
    └── baz.txt
$ \ls -l
total 0
drwxr-xr-x. 2 username username 40 Jul 16 10:03 bar
-rw-r--r--. 1 username username  0 Jul 16 10:03 baz.log
drwxr-xr-x. 3 username username 80 Jul 16 10:03 foo
```
the field between `username` and `Jul 16` doesn't provide the number of files/subdirs if I am not mistaken. 
I checked the manpage of `ls` and was stunned to find **NO** hint to what `ls -l` actually prints out. :-1: 

A quote from `man ls` on my system (fc31, kernel `5.7.7-100.fc31.x86_64`):
```
       -l     use a long listing format
```

**Review by psteinb — CHANGES_REQUESTED** [review]

File: _episodes/05-scripts.md (hunk 3 above)
Comment (tkphd) [reply _episodes/05-scripts.md]: You're absolutely right.

File: _episodes/05-scripts.md (hunk 5 above)
Comment (tkphd) [reply _episodes/05-scripts.md:114]: Windows users would be more familiar with "shortcuts" than symbolic or hard links.

File: _episodes/05-scripts.md (hunk 6 above)
Comment (tkphd) [reply _episodes/05-scripts.md]: [IBM](https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.1.0/com.ibm.zos.v2r1.bpxa500/lscmd.htm#lscmd__longf) indicates it's the *number of filesystem blocks* occupied by the contents of the directory.

File: _episodes/05-scripts.md (hunk 5 above)
Comment (psteinb) [reply _episodes/05-scripts.md:114]: ah, true. You are right.

File: _episodes/05-scripts.md (hunk 6 above)
Comment (psteinb) [reply _episodes/05-scripts.md]: Ok, should we add that here or is that too much mental overhead? Balancing completeness versus simplicity.

Comment (tkphd) [reply _episodes/05-scripts.md]: ~Yes, please add that here :-)~ Changed (503362a) to explain more accurately, if less concisely. Clarity edits are welcome.

**Review by psteinb — APPROVED** [review]
Super cool. I like it.

**Pull request merged**